chat-with-rendezvous/chat.go

```diff
@@ -86,26 +110,49 @@ func main() {
 
 	ctx := context.Background()
 
+	// Configure p2p host
+	privkey, _, err := crypto.GenerateKeyPair(crypto.RSA, 1024)
+	if err != nil {
+		panic(err)
+	}
+	transports := libp2p.DefaultTransports
+	addrs := make([]multiaddr.Multiaddr, len(config.ListenAddresses))
+	copy(addrs, config.ListenAddresses)
+
+	options := []libp2p.Option{transports, libp2p.DefaultMuxers}
+	options = append(options, libp2p.ListenAddrs(addrs...))
+	options = append(options, libp2p.Identity(privkey))
```

Comment:
here again, all this dance is unnecessary -- just let libp2p configure the transports, muxers, and identity generation.

Reply:
Changed to default